src/utils/tablify.js

```diff
@@ -33,11 +33,24 @@ const chars = {
 	middle: '│',
 };
 
+const errorCyclicObject = 'Error: cyclic object cannot be displayed.';
+
 const getNestedValue = data => keyString => keyString.split('.').reduce((obj, key) => obj[key], data);
 
 const addValuesToTable = (table, data) => {
 	const nestedValues = table.options.head.map(getNestedValue(data));
-	const valuesToPush = nestedValues.map(value => (Array.isArray(value) ? value.join('\n') : value));
+	const valuesToPush = nestedValues.map((value) => {
+		if (Array.isArray(value)) {
+			return value.join('\n');
+		} else if (value && typeof value === 'object') {
```

Comment:
The `value &&` part is because of `null`?

Reply:
yes! it's because of `null`